addon/components/-ember-popper-legacy.js

```diff
@@ -11,12 +15,36 @@ export default class EmberPopper extends EmberPopperBase {
   didInsertElement() {
     this._initialParentNode = this.element.parentNode;
 
+    if (!GTE_EMBER_1_13) {
+      addObserver(this, 'renderInPlace', this, this.didUpdateAttrs);
```

Comment:
Tests failed on 1.11 because `didUpdateAttrs` doesn't work yet 😭 

This adds observers conditionally in 1.11 only